content/docs/user-guide/visualization/animation/using-actor-LODs-optimize-game-performance.md

```diff
@@ -1,107 +1,178 @@
 
-**To use the FBX Settings to check LODs**
+### Manually set skeleton LODs
 
-1. In O3DE Editor, choose **Tools**, **Asset Browser**.
 
-1. Right click the `.fbx` file and choose **Edit Settings**. The **FBX Settings** tool shows a **Level of Detail** modifier that lists the additional LOD meshes.
+7. In Scene Settings, select the **Actors** tab.
 
-    ![LOD entries in the FBX Settings tool](/images/user-guide/character-fbx-settings-lod-optimize-1.png)
+8. Choose **Add Modifier** and select **Skeleton LOD** from the modifier list.
 
-You can use the **FBX Settings** tool to verify that your LODs were imported correctly. Currently, you can also unselect a mesh or skeleton for each LOD, but you can't move a mesh or skeleton across a level or add a new mesh or skeleton to a level.
+9. In the Skeleton LOD modifier, choose the {{< icon "add.svg" >}} to add a skeleton LOD. Add as many skeleton LODs as required.
 
-## Adding Actor LOD to a Level in O3DE
+10. For each skeleton LOD, choose the {{< icon "browse-edit-select-files.svg" >}} node select button to open the scene node list, and select the bone nodes to include in the LOD.
 
-After you import your LODs into O3DE Editor, you can add an actor LOD to a level.
+{{< note >}}
+You aren't required to build or manually set-up skeleton LODs to use actor LODs. In most scenarios, an actor can use the same base skeleton for each LOD, or optimized skeletons can be automatically generated for each LOD based on skin weight information. It's only recommended to manually assign skeleton LODs in scenarios where LodGroups or automatic skeleton LOD generation does not yield desired results.
+{{< /note >}}
 
-**To add an actor LOD to a level in O3DE**
+When you've assigned all the meshes and bones to the appropriate LODs, choose **Save** at the bottom of the Scene Settings window to save your preferences and automatically process the actor with its LODs.
 
-1. In O3DE Editor, choose **Tools**, **Asset Browser**.
+## Automatic LOD setup with LodGroups
 
-1. Select and drag the exported `.fbx` file to the viewport. In the **[Actor](/docs/user-guide/components/reference/animation/actor/)** component that appears in the **Entity Inspector**, a shared material file with multiple materials contains the LOD asset.
+LodGroups are created in a digital content creation (DCC) application such as Maya or Blender by arranging the meshes and skeletons for each LOD in a hierarchy of groups. LodGroups are automatically processed and don't require additional configuration in Scene Settings.
 
-1. To preview the LODs, change the number for **LOD Level** in the **Actor** component.
+An LodGroup node specifies that its child groups each contain an LOD. Maya has a special LOD Group tool that can be accessed from the edit menu to create LodGroups. In other applications, such as Blender, you can create LodGroups by doing the following:
```

Comment:
Might consider linking it to Blender. Chose this instance of it because this is a "first" reference of it in context with the following example. 
```suggestion
An LodGroup node specifies that its child groups each contain an LOD. Maya has a special LOD Group tool that can be accessed from the edit menu to create LodGroups. In other applications, such as [Blender](https://www.blender.org/), you can create LodGroups by doing the following:
```